{
   "CVE_data_meta" : {
      "ASSIGNER" : "cve@mitre.org",
      "ID" : "CVE-2011-4137",
      "STATE" : "PUBLIC"
   },
   "affects" : {
      "vendor" : {
         "vendor_data" : [
            {
               "product" : {
                  "product_data" : [
                     {
                        "product_name" : "n/a",
                        "version" : {
                           "version_data" : [
                              {
                                 "version_value" : "n/a"
                              }
                           ]
                        }
                     }
                  ]
               },
               "vendor_name" : "n/a"
            }
         ]
      }
   },
   "data_format" : "MITRE",
   "data_type" : "CVE",
   "data_version" : "4.0",
   "description" : {
      "description_data" : [
         {
            "lang" : "eng",
            "value" : "The verify_exists functionality in the URLField implementation in Django before 1.2.7 and 1.3.x before 1.3.1 relies on Python libraries that attempt access to an arbitrary URL with no timeout, which allows remote attackers to cause a denial of service (resource consumption) via a URL associated with (1) a slow response, (2) a completed TCP connection with no application data sent, or (3) a large amount of application data, a related issue to CVE-2011-1521."
         }
      ]
   },
   "problemtype" : {
      "problemtype_data" : [
         {
            "description" : [
               {
                  "lang" : "eng",
                  "value" : "n/a"
               }
            ]
         }
      ]
   },
   "references" : {
      "reference_data" : [
         {
            "url" : "http://openwall.com/lists/oss-security/2011/09/11/1"
         },
         {
            "url" : "http://openwall.com/lists/oss-security/2011/09/13/2"
         },
         {
            "url" : "http://openwall.com/lists/oss-security/2011/09/15/5"
         },
         {
            "url" : "https://bugzilla.redhat.com/show_bug.cgi?id=737366"
         },
         {
            "url" : "https://www.djangoproject.com/weblog/2011/sep/09/"
         },
         {
            "url" : "https://www.djangoproject.com/weblog/2011/sep/10/127/"
         },
         {
            "url" : "http://www.debian.org/security/2011/dsa-2332"
         },
         {
            "url" : "https://hermes.opensuse.org/messages/14700881"
         },
         {
            "url" : "http://secunia.com/advisories/46614"
         }
      ]
   }
}
